submissions/israelOladosu/introductionToHtml/index.html

```diff
@@ -1,72 +1,121 @@
-</html><!DOCTYPE html>
+</html>
+<!DOCTYPE html>
 <html lang="en">
+
 <head>
   <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
   <title>Farmville</title>
+  <link rel="stylesheet" href="../css/product.css">
 </head>
+
 <body>
-   <header>
+  <header>
     <!-- container for the site logo -->
-    <div>
-     <a href="index.html">
-      <img src="./img/Farmville flat green.png" alt="Farmville">
-     </a>
+    <div class="logo">
+      <a href="index.html">
+        <img src="./img/Farmville white logo.png" alt="Farmville">
```

Comment:
Update the name here as well

Reply:
Thank you.

I have made the requested changes